### PR TITLE
delete button for ideas

### PR DESCRIPTION
**Pull request by NinaBusz**
- reverse ideas-list on new page (newest idea on top)
- delete button for ideas
- navbar for navigation and a bit styling

### CODE REVIEW

File: components/Idea/Idea.js [hunk 1]
```diff
@@ -1,8 +1,16 @@
-export default function Idea({ id, date, text }) {
+export default function Idea({ id, date, text, handleDeleteIdea }) {
   return (
-    <li className="newIdea">
-      <small>{date}</small>
+    <li className="ideasList__newIdea">
       <div>{text}</div>
```
Comment (stefankorder) [inline components/Idea/Idea.js]: A `<p>` tag instead of a `<div>` tag would be better.

File: pages/ideasPage/index.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    localStorage.setItem("ideasData", JSON.stringify(ideas));
+  }, [ideas]);
+
+  const deleteIdea = (id) => {
+    const savedIdeas = ideas.filter((idea) => idea.id !== id);
+    setIdeas(savedIdeas);
+  };
+
+  return (
+    <div className="ideasList">
```
Comment (stefankorder) [inline pages/ideasPage/index.js]: Use an `<article>` instead of an `<div>`.

File: components/Idea/Idea.js [hunk 3]
```diff
@@ -1,8 +1,16 @@
...
+      <div className="ideasList__newIdea--bottom">
+        <small>{date}</small>
+        <button
+          className="ideasList__newIdea--deleteButton"
+          onClick={() => handleDeleteIdea(id)}
+        >
+          Löschen
+        </button>
+      </div>
```
Comment (stefankorder) [inline components/Idea/Idea.js]: Use an `<article>` instead of a `<div>`.

**Review by stefankorder — CHANGES_REQUESTED** [review]
LGTM! Just a few changes i would recommend you.

File: components/Idea/Idea.js (hunk 3 above)
Comment (NinaBusz) [reply components/Idea/Idea.js]: Thank you! I changed the tags :)

**Pull request merged**